archunit/src/main/java/com/tngtech/archunit/core/domain/AnnotationPropertiesFormatter.java

```diff
@@ -114,8 +132,21 @@ public String apply(String input) {
             return this;
         }
 
-        AnnotationValueFormatter build() {
-            return new AnnotationValueFormatter(this);
+        /**
+         * Configures that the identifier is omitted if the annotation is a
+         * <a href="https://docs.oracle.com/javase/specs/jls/se13/html/jls-9.html#jls-9.7.3">single-element annotation</a>
```

Comment:
The PR for JDK14 refers to Java SE13? 😜

Reply:
I tried to be consistent with other references to the JLS like in `JavaClass` 😜 
https://github.com/TNG/ArchUnit/blob/1b3a73fd935032869642ca78a20304040da0340a/archunit/src/main/java/com/tngtech/archunit/core/domain/JavaClass.java#L263-L264

Reply:
I'm afraid this is mostly inconsistent anyway :thinking: Maybe we should at some point update all the links to the current docs... For now, the link for JDK 14 is almost the same, I just used that one.